app/controllers/api/v1/sources_controller.rb

```diff
@@ -42,7 +42,8 @@ def check_availability
 
       def check_application_availability(source)
         source.application_types.each do |app_type|
-          url = app_type.availability_check_url
+          app_env_prefix = app_type.name.split('/').last.upcase.tr('-', '_')
```

Comment:
Hi @agrare, code driven based on name, let me know if you prefer a different method.

Reply:
:+1: